testdata/reconcile/test_ofx_matching/0/candidates.diff

```diff
@@ -0,0 +1,3 @@
+<journal-dir>/journal.beancount
++
++2018-07-03 price TYCDT                  1.554232804232804232804232804 USD
```

Comment:
In another PR someone should think carefully about how to truncate numbers like this correctly. They are an eye sore.